bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafControllerHandler.java

```diff
@@ -119,6 +117,14 @@ public void initialize() {
         setRefreshIntervall(config.refreshInterval);
         setAuthToken(config.authToken);
 
+        @Nullable String property = getThing().getProperties().get(Thing.PROPERTY_MODEL_ID);
+        if (property.equals(MODEL_ID_CANVAS)) {

```

Comment:
Position literals first in `String` comparisons - that way if the `String` is `null` you won't get a `NullPointerException`, it'll just return `false`. [source: PMD](http://pmd.sourceforge.net/pmd-4.3.0/rules/design.html#PositionLiteralsFirstInComparisons)

Check others :-)